CHANGELOG.rst

```diff
@@ -2,6 +2,42 @@
 CHANGELOG
 =========
 
+2.9.0
+=====
+
+**ENHANCEMENTS**
+
+* Add support for multiple queues and multiple instance types feature with the Slurm scheduler.
+* Extend NICE DCV support to ARM instances.
+* Extend support to disable hyperthreading on instances (like \*.metal) that don't support CpuOptions in
+  LaunchTemplate.
+* Enable support for NFS 4 for the filesystems shared from the head node.
+* Add CLI utility to convert configuration files with Slurm scheduler to new format to support multiple queues
+  configuration.
+* Add script wrapper to support Torque-like commands with the Slurm scheduler.
+
+**CHANGES**
+
+* Introduce new configuration sections and parameters to support multiple queues and multiple instance types.
+* Optimize scaling logic with Slurm scheduler, no longer based on Auto Scaling groups.
+* A Route53 private hosted zone is now created together with the cluster and used in DNS resolution inside cluster nodes
+  when using Slurm scheduler.
+* Upgrade Slurm to version 20.02.4.
+* Upgrade NICE DCV to version 2020.1-9012.
+* Use private ip instead of master node hostname when mounting shared NFS drives.
+* Add new log streams to CloudWatch: chef-client, clustermgtd, computemgtd, slurm_resume, slurm_suspend.
+* Remove dependency on cfn-init in compute nodes bootstrap.
```

Comment:
I'd say a word about avoiding throttling  on CloudFormation and delays when a large number of compute  nodes join the cluster at once. 
And in this sense, it's an enhancement